packages/core/src/batch/BatchSystem.ts

```diff
@@ -74,7 +79,7 @@ export class BatchSystem extends System
      * @param arr
      * @param maxTextures
      */
-    copyBoundTextures(arr, maxTextures)
+    copyBoundTextures(arr: Array<BaseTexture>, maxTextures: number): void
```

Comment:
Why `Array<BaseTexture>` and not `BaseTexture[]` (or why the latter format used in different places)? Again may be a good idea to enforce one standard via eslint.

Reply:
linting rulle is good idea.
With autfix of course ;-)

Reply:
yeah, maybe use [] there because MAX_TEXTURES is a constant :)

Reply:
@eXponenta so I understand it's a convention used in Pixi's codebase? That sounds like something that could be written down, unless it is and I never saw it :).

Reply:
I wrote it down in this PR but we really need a place to write those things.